src/osd/ECBackend.cc

```diff
@@ -902,7 +902,16 @@ void ECBackend::handle_sub_write(
     }
   }
   clear_temp_objs(op.temp_removed);
-  dout(30) << __func__ << " missing before " << get_parent()->get_log().get_missing().get_items() << dendl;
+  get_parent()->log_operation(
```

Comment:
Is there is reason why we want to move the dout of "missing before"?

Reply:
dropped that cosmetic change ( I was trying to wrap that line into 80 chars, but I think it is fine to leave as it is...)